lib/helpers/moduleHelper.js

```diff
@@ -155,6 +155,61 @@ var helper = {
       return Promise.reject(e);
     }
     return Promise.resolve(result);
+  },
+
+
+  /**
+   * Escape HTML string chars
+   * Based on component/escape-html
+   * @param {String} string
+   * @returns {String}
+   */
+  escape (string) {
```

Comment:
Why implementing this function ourselves when we could just use https://lodash.com/docs#escape?